include/form_functions.js

```diff
@@ -31,8 +32,18 @@ function closeAddStudentForm(){
 //Add credits form   
 // --------------------------------------------------------------------------
 function openCreditForm(buttonclicked, credits){ 
-    document.getElementById('hiddenValue').value = buttonclicked;                             //I'm setting the id parameter to the type hidden field so I can know wich button was clicked in the Request
+    document.getElementById('hiddenButtonName').value = buttonclicked;                             //I'm setting the id parameter to the type hidden field so I can know wich button was clicked in the Request
     
+    //AJAX request to get the student from the DB
```

Comment:
I put this here because I was thinking about doing AJAX request instead of passing the credits, (so I was about to work with this code as template) but then I realized that the remaining credits are being displayed in the page anyway so I just keep it like this and then I deleted this code (You will see it in the following commit). Sorry for any confusion. I didn't realized I committed with this here